extensions/job_management/standard/requirements/job-management/update/REQ_response-locked.adoc

```diff
@@ -0,0 +1,9 @@
+[[req_job-management_update_response-locked]]
+[requirement]
+====
+[%metadata]
+label:: /req/job-management/update-response-locked
+part:: If a job is locked, meaning that it is currently being processed (status set to `accepted` or `running`), the server SHALL respond with HTTP status code `423 Locked`.
```

Comment:
in openEO that's queued or running, we also include an error message in the JSON payload.